homeassistant/components/nextbus/sensor.py

```diff
@@ -67,26 +88,27 @@ def validate_tags(client, agency, route, stop):
     return True
 
 
-def setup_platform(
+async def async_setup_entry(
     hass: HomeAssistant,
-    config: ConfigType,
-    add_entities: AddEntitiesCallback,
-    discovery_info: DiscoveryInfoType | None = None,
+    config: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Load values from configuration and initialize the platform."""
-    agency = config[CONF_AGENCY]
-    route = config[CONF_ROUTE]
-    stop = config[CONF_STOP]
-    name = config.get(CONF_NAME)
-
     client = NextBusClient(output_format="json")
```

Comment:
This should probably be moved to `__init__.py`, and made available inside `hass["data"]`.
Right now you have only one platform but maybe in the future you will have more.

Reply:
The client has been moved in the Data Coordinator PR #92218